src/nativeAssetManager.js

```diff
@@ -190,6 +204,19 @@ export function newNativeAssetManager(win) {
     return placeholders;
   }
 
+  /*
+   * Searches the DOM for existence passed string
+   */
+  function scanForString(str) {
+    const strIndex = win.document.body.innerHTML.indexOf(str);
+
+    if (~strIndex) {
+      return true;
+    }
+
+    return false;
+  }
```

Comment:
Could just do

```javascript
return win.document.body.innerHTML.indexOf(str) !== -1
```


Not sure this function is even being used?

Reply:
I dont remember this function but cant find a use for it so removing